www/include/configuration/configObject/contact/DB-Func.php

```diff
@@ -1038,3 +833,146 @@ function getContactIdByName($name)
+                break;
+            case 'contact_oreon':
+                 // ldap import, then force contact to be a user
+                if (isset($_POST['contact_select']['select'])) {
+                    $bindParams[$inputName] = '1';
+                } else {
+                    $bindParams[$inputName] =
+                        in_array($inputValue[$inputName], ['0', '1'])
+                            ? $inputValue[$inputName]
+                            : null;
+                }
+                break;
+            case 'contact_activate':
+                $bindParams[$inputName] =
+                    in_array($inputValue[$inputName], ['0', '1'])
+                        ? $inputValue[$inputName]
+                        : null;
+                break;
+            case 'reach_api':
+            case 'reach_api_rt':
+                $bindParams[$inputName] =
+                    in_array($inputValue[$inputName], ['0', '1'])
+                        ? (int) $inputValue[$inputName]
+                        : 0;
+                break;
+            case 'contact_enable_notifications':
+                $bindParams[$inputName] =
+                    in_array($inputValue[$inputName], ['0', '1', '2'])
+                        ? $inputValue[$inputName]
+                        : '2';
+                break;
+            case 'contact_admin':
+                $bindParams[$inputName] =
+                    in_array($inputValue[$inputName], ['0', '1'])
+                        ? $inputValue[$inputName]
+                        : '0';
+                break;
+            case 'contact_type_msg':
+                $bindParams[$inputName] =
+                    in_array($inputValue, ['txt', 'html', 'pdf'])
+                        ? $inputValue
+                        : 'txt';
+                break;
+            case 'contact_passwd':
+                if (!empty($inputValue)) {
+                    $password = $utilsObject->encodePass(
+                        $inputValue,
+                        $encryptType == 2 ?  'sha1' : 'md5'
+                    );
+                    $bindParams[$inputName] = $password;
+                }
+                break;
+            case 'contact_lang':
+                if (!empty($inputValue)) {
+                    $bindParams[$inputName] =
+                        (($inputValue = filter_var($inputValue, FILTER_SANITIZE_STRING)) === false)
+                            ? 'browser'
+                            : $inputValue;
+                }
+                break;
+            case 'contact_auth_type':
+                if (!empty($inputValue)) {
+                    $bindParams[$inputName] =
+                        (($inputValue = filter_var($inputValue, FILTER_SANITIZE_STRING)) === false)
+                            ? 'local'
+                            : $inputValue;
+                }
+                break;
+            case 'contact_name':
+            case 'contact_alias':
```

Comment:
could these, be empty ?

Reply:
https://github.com/centreon/centreon/pull/9624/files#diff-4cfd7b0df58e097781835a57f629acf237f44234e5f7f03db093d83ba7808d24R968